src/test/java/org/jenkinsci/plugins/workflow/WorkflowRunTest.java

```diff
@@ -276,4 +277,21 @@ public void failedToStartRun() throws Exception {
         assertEquals(Collections.emptyList(), iba.getCauses());
     }
 
+    @Issue("JENKINS-34547")
+    @LocalData
+    @Test public void concurrentBuildsMigrationOnByDefault() throws Exception {
```

Comment:
:ant: the code in question is in `WorkflowJob`, so it would make more sense for this to be in a `WorkflowJobTest`. Or in a `DisableConcurrentBuildsJobPropertyTest`.